test/rating.js

```diff
@@ -16,40 +16,13 @@ describe('A track in Google Music', function () {
   });
   browserUtils.execute(function playViaApi () {
     window.googleMusic.playback.playPause();
+    window.googleMusic.playback.toggleRepeat();
```

Comment:
If it's to keep on playing a track after the thumbs down, then it might fail if we hit `toggleRepeat` into a no-repeat. Here are some other solutions:

- Write a `setRepeat` method and use that (hooray for inconsistencies in the original API \o/)
- Run `playAnything` in a `before` for each of these tests